test/e2e/run_entrypoint_test.go

```diff
@@ -46,6 +44,7 @@ CMD []
 	})
 
 	It("podman run entrypoint", func() {
+		SkipIfRemote()
```

Comment:
I'm not sure why this and the subsequent ones are skipped? They're all tagged as `foobar.com/something` so the `localhost` issue shouldn't apply. And I don't understand what you mean by `localhost` anyway, I'm playing with podman-remote on my system (with master) and am not having problems building, tagging, or running images with unqualified names...?

Reply:
I just wanted to get as many in as possible.  I did not review why they were blowing up. I was hoping to assign each one to others who are closer to the issue, to investigate dropping the SkipIfRemote().